src/main/resources/application.properties

```diff
@@ -137,3 +137,13 @@ management.health.elasticsearch.enabled=false
 management.health.status.order=DOWN, OUT_OF_SERVICE, UNKNOWN, UP, OUT_OF_SYNC, PROBLEM, COULD_NOT_CONNECT
 
 spring.banner.location=classpath:ccd-data-store-banner.txt
+
+# OAuth2
+
+ccd.oauth2.client_id: ${OAUTH2_CLIENT_ID:ccd_gateway}
+ccd.oauth2.client_secret: ${OAUTH2_CLIENT_SECRET:OOOOOOOOOOOOOOOO}
+ccd.oauth2.redirect_url: ${OAUTH2_REDIRECT_URL:http://localhost:3451/oauth2redirect}
+ccd.cache.warm.up.enabled: ${CCD_CACHE_WARM_UP_ENABLED:false}
```

Comment:
where are we overriding this default value for Demo and AAT to be true?

Reply:
Done.